client/src/modules/dashboard/Chapters/Users/pages/ChapterUsersPage.tsx

```diff
@@ -191,26 +192,48 @@ export const ChapterUsersPage: NextPageWithLayout = () => {
                       Change
                     </Button>
                   )}
-                  {is_bannable &&
-                    (bans.has(otherUser.id) ? (
+                  {bans.has(otherUser.id) ? (
+                    <Tooltip
+                      hasArrow
+                      label={
+                        !is_bannable &&
+                        `You can't unban this member, due to missing privileges in the Instance`
```

Comment:
```suggestion
                        `You can't unban other administrators.`
```
If you're an owner, you won't see this message.
If you're banned, you won't see this page.

So, the only people that will see this are unbanned administrators.